Source/SwiftLintFramework/Models/SwiftVersion.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SwiftVersion.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 12/29/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+enum SwiftVersion {
+    case two
+    case three
+
+    static let current: SwiftVersion = {
+        let file = File(contents: "#sourceLocation()")
```

Comment:
We should propose that a request type be added to SourceKit to get the current Swift version. But this is good for now.